Topic3TDD/src/exc2/WordWrapperTest.java

```diff
@@ -0,0 +1,44 @@
+package exc2;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Test;
+
+public class WordWrapperTest {
+
+	@Test
+	public void firstExampleTest() {

```

Comment:
It's ok, but a good practice in tests is to try to declare in the name of the method what are you testing for example testWrapperStringShorterThanRowLength